### PR TITLE
Add a reset configuration feature service launcher

### LINKED ISSUE #266: Add a reset configuration feature at the service launcher
Add a reset configuration button at the service configuration launcher : 

Capture d’écran 2021-10-25 à 10 18 10

### PR DESCRIPTION
**Pull request by smith-julien**
Add a reset configuration feature at the service launcher #266 

close #266

### CODE REVIEW

File: src/ui/components/pages/Catalog/CatalogLauncher/CatalogLauncher.tsx [hunk 1]
```diff
@@ -328,6 +340,8 @@ export const CatalogLauncher = memo((props: Props) => {
     );
 });
 
+CatalogLauncher.displayName = CatalogLauncher.name;
```
Comment (garronej) [inline src/ui/components/pages/Catalog/CatalogLauncher/CatalogLauncher.tsx]: While it's good practice to give a displayName this won't work, the `CatalogLauncher` has no name (it's an arrow function).  

This is what you should do instead:  

`CatalogLauncher.displayName = symToStr({ CatalogLauncher });`  

and import `symToStr` like:  

```ts
import { symToStr } from "tsafe/symToStr";
```

File: src/ui/components/pages/Catalog/CatalogLauncher/CatalogLauncherConfigurationCard.tsx [hunk 2]
```diff
@@ -124,6 +124,8 @@ export const CatalogLauncherConfigurationCard = memo(
     }
 );
 
+CatalogLauncherConfigurationCard.displayName = CatalogLauncherConfigurationCard.name;
```
Comment (garronej) [inline src/ui/components/pages/Catalog/CatalogLauncher/CatalogLauncherConfigurationCard.tsx]: Same

File: src/ui/components/pages/Catalog/CatalogLauncher/CatalogLauncherMainCard.tsx [hunk 3]
```diff
@@ -144,6 +152,8 @@ export const CatalogLauncherMainCard = memo((props: Props) => {
     );
 });
 
+CatalogLauncherMainCard.displayName = CatalogLauncherMainCard.name;
```
Comment (garronej) [inline src/ui/components/pages/Catalog/CatalogLauncher/CatalogLauncherMainCard.tsx]: Same

File: src/ui/components/pages/Catalog/CatalogLauncher/CatalogLauncher.tsx (hunk 1 above)
Comment (smith-julien) [reply src/ui/components/pages/Catalog/CatalogLauncher/CatalogLauncher.tsx]: thx, I was looking for this function 😄

**Pull request merged**